vipps-psp-api.md

```diff
@@ -281,7 +281,16 @@ NB Vipps will communicate more regarding Mastercard D-SCA shortly.
 
 ### Token Requestor Ids
 
-For Visa/Mastercard the Trid is an eleven digit number. And is exposed for reference in the API.
+For Visa/Mastercard the TRID is an eleven digit number. And is added by the Scheme in the processing of the Payment.
+
+### Magic Numbers for EMVCo Tokens
+
+Any request to the v3 API will return a Visa Token. However this can be changed by setting the amount in the init request. No matter what is selected in the app the Token returned in the MakePayment request will be:
```

Comment:
Maybe chang to "Requsets to the v3 API will return a Visa Token by default, however…"